### ISSUE TITLE
Twilio::TwiML::MessagingResponse message body XML is broken

### ISSUE THREAD

**Issue by jamby** [twilio/twilio-ruby#347]
Body is not being appending inside the `Message` XML, it's added on as an attribute. This works fine in 5.2.0, but not 5.2.3.

**Version:** 5.2.3

### Code Snippet
```ruby
response = Twilio::TwiML::MessagingResponse.new
response.message(body: 'Store Location: 123 Easy St.')

puts response
```

### Exception/Log
XML is:
```xml
<?xml version="1.0" encoding="UTF-8"?><Response><Message body="Store Location: 123 Easy St."/></Response>
```

should be:
```xml
<?xml version="1.0" encoding="UTF-8"?><Response><Message>Store Location: 123 Easy St.</Message></Response>
```

### Steps to Reproduce
1. Open ruby console on latest version, 5.2.3
2. Paste code from https://www.twilio.com/docs/api/twiml/sms/message
3. Notice XML is wrong

**Comment by Shivang44** [comment]
I'm new to Twilio SMS so I may be wrong, but I believe they updated the API so it's "message" not "body". This works for me: 

response.message(message: "Store Location: 123 Easy St.")


It's frustrating, they have 3 different pages telling you to do it 3 different ways, but as of 9/30/2017, I believe that is the way to do it.

**Comment by jamby** [comment]
Twilio might have updated their API docs since this issue, because I lifted that code straight from the docs.

Good to hear they updated it, but that's weird that they're changing the API more than they did from version 4 to 5.

**Comment by efossier** [comment]
@jamby @Shivang44 Sorry for the confusion, in this case the docs are actually out of date. We recently changed this param to be `body` while simultaneously adding support for passing arbitrary kwargs which show up as attributes on the xml element... We should have communicated this better internally and I apologize for the conflicting information you've found so far.

Therefore the right keyword argument to pass to set the body is `body`. I've flagged the out of date doc with the docs team and we'll get that fixed asap. The code sample in the original post is accurate.
